src/Draco.Compiler/Internal/Symbols/FunctionTypeSymbol.cs

```diff
@@ -19,6 +20,8 @@ internal sealed class FunctionTypeSymbol : TypeSymbol
 
     public override Symbol? ContainingSymbol => null;
 
+    public override IEnumerable<Symbol> Members => this.ReturnType.Members;
```

Comment:
Let's not propagate members here, as it's incorrect to say that the function type has these members